.travis.yml

```diff
@@ -27,9 +27,8 @@ install:
 script:
   - set -e
   - ls -lah
-  - docker build -t sqlflow:dev -f Dockerfile.dev .
-  - docker run --rm -v $GOPATH/src:/go/src -w /go/src/github.com/sql-machine-learning/sqlflow sqlflow:dev pre-commit run -a
-  - docker run --rm -v $GOPATH:/go -w /go/src/github.com/sql-machine-learning/sqlflow sqlflow:dev bash scripts/test.sh
+  - docker run --rm -v $GOPATH/src:/go/src -w /go/src/github.com/sql-machine-learning/sqlflow sqlflow/sqlflow:dev pre-commit run -a
```

Comment:
I put compiling SQLFlow develop branch into the `Dockerfile` so that SQLFlow binaries are located in the docker image. If do `docker build` in CI then, it's actually building the code twice, one for develop branch, one for current PR branch. Thought would omit one.

Reply:
I added `docker build` back since we need to also test `Dockerfile` during pull requests.